src/Strategy/CommonClassesStrategy.php

```diff
@@ -53,19 +53,19 @@ final class CommonClassesStrategy implements DiscoveryStrategy
      */
     private static $classes = [
         MessageFactory::class => [
-            ['class' => NyholmHttplugFactory::class, 'condition' => [NyholmHttplugFactory::class]],
+            ['class' => NyholmPsr17Factory::class, 'condition' => [NyholmPsr17Factory::class]],
```

Comment:
in the interest of BC with older nyholm libraries, should we keep the legacy factory as second option, after the psr17 factory?

Reply:
ah okay, https://github.com/Nyholm/psr7/blob/1.0.0/src/Factory/Psr17Factory.php says the factory is available in all versions of nyholm psr7